dbt_sugar/core/task/base.py

```diff
@@ -69,29 +69,96 @@ def get_not_documented_columns(
                         not_documented_columns[column["name"]] = COLUMN_NOT_DOCUMENTED
         return not_documented_columns
 
+    def __combine_two_list_without_duplicates(
+        self, list1: List[Any], list2: List[Any]
+    ) -> List[Any]:
+        """
+        Method to combine two list without duplicates.
+
+        :param list1: Optional[List[Any]]
+        :param list2: List[Any]
+        :return: List[Any]
```

Comment:
we use google docstrings in this project, you should be able to configure this in your IDE
the google dosctrings look like this:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html